pkg/flowkit/services/transactions_test.go

```diff
@@ -183,4 +186,61 @@ func TestTransactions_Integration(t *testing.T) {
 
 	})
 
+	t.Run("Build Transaction with Imports", func(t *testing.T) {
+		t.Parallel()
+		state, s := setupIntegration()
+		setupAccounts(state, s)
+
+		srvAcc, _ := state.EmulatorServiceAccount()
+		signer := srvAcc.Address()
+
+		// setup
+		c := config.Contract{
+			Name:    tests.ContractHelloString.Name,
+			Source:  tests.ContractHelloString.Filename,
+			Network: "emulator",
+		}
+		state.Contracts().AddOrUpdate(c.Name, c)
+
+		n := config.Network{
+			Name: "emulator",
+			Host: "127.0.0.1:3569",
+		}
+		state.Networks().AddOrUpdate(n.Name, n)
+
+		d := config.Deployment{
+			Network: n.Name,
+			Account: srvAcc.Name(),
+			Contracts: []config.ContractDeployment{{
+				Name: c.Name,
+				Args: nil,
+			}},
+		}
+		state.Deployments().AddOrUpdate(d)
+		_, _ = s.Accounts.AddContract(srvAcc, tests.ContractHelloString.Name, tests.ContractHelloString.Source, false)
+
+		tx, err := s.Transactions.Build(
+			signer,
+			[]flow.Address{signer},
+			signer,
+			srvAcc.Key().Index(),
+			tests.TransactionImports.Source,
+			tests.TransactionImports.Filename,
+			1000,
+			nil,
+			n.Name,
+		)
+
+		assert.NoError(t, err)
+		ftx := tx.FlowTransaction()
+		assert.Equal(t,
+			string(ftx.Script),
+			strings.ReplaceAll(
+				string(tests.TransactionImports.Source),
+				"import Hello from \"./contractHello.cdc\"",
+				"import Hello from 0xf8d6e0586b0a20c7",
```

Comment:
Is this address a constant somewhere? Hard to know if this the correct address otherwise